### PR TITLE
getTransientData function

### PR DESCRIPTION
**Pull request by loganallengray**
Ticket #32 
I added the getTransientData function in database.js, which is needed for multiple modules.

### CODE REVIEW

File: scripts/FacilityInventories.js [hunk 1]
```diff
@@ -1,8 +1,14 @@
-import { getFacilityInventories } from "./database.js";
+import { getFacilityInventories, getFacilities, get } from "./database.js";
```
Comment (will-hedges) [inline scripts/FacilityInventories.js:1]: do we need this last `get` ?

File: scripts/database.js [hunk 2]
```diff
@@ -39,7 +39,7 @@ const database = {
         { id: 4, name: "Titan" },
         { id: 5, name: "Proteus" },
     ],
-    transientState: {},
+    transientState: {test: "teehee"},
```
Comment (will-hedges) [inline scripts/database.js:42]: should this stay in?

**Review by thegoodcorg — APPROVED** [review]
👍

**Review by bigjlr2004 — APPROVED** [review]
looks good

**Review by will-hedges — APPROVED** [review]

**Pull request merged**